### PR TITLE
Chore/deprecate functions store

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/components/interfaces/Database/Functions/CreateFunction.tsx [hunk 1]
```diff
@@ -27,7 +26,7 @@ class CreateFunctionFormState {
   // @ts-ignore
   args: { value: { name: string; type: string; error?: string }[] }
   // @ts-ignore
-  behavior: { value: string; error?: string }
+  behaviour: { value: string; error?: string }
```
[the remark this answers is not in the document]
Comment (joshenlim) [reply apps/studio/components/interfaces/Database/Functions/CreateFunction.tsx:29]: welp thats a spelling error on API end 😂

[the remark this answers is not in the document]
Comment (joshenlim) [reply apps/studio/components/interfaces/Database/Functions/CreateFunction.tsx:29]: im actually confused how this was working before

[the remark this answers is not in the document]
Comment (joshenlim) [reply apps/studio/components/interfaces/Database/Functions/CreateFunction.tsx:29]: wait sorry i messed up here

**Review by alaister — APPROVED** [review]
LGTM! Just a couple small ones:

**Pull request merged**